indico/modules/auth/blueprint.py

```diff
@@ -37,6 +38,8 @@
 
 _bp.add_url_rule('/reset-password/', 'resetpass', RHResetPassword, methods=('GET', 'POST'))
 
+_bp.add_url_rule('/sessions/invalidate', 'sessions_invalidate', RHSessionsInvalidate, methods=('POST',))
```

Comment:
I think this should be within tthe next block - it makes perfect sense for an admin to perform this on another user.